jsonexample/android/app/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "dev.flutter.example.jsonexample"
```

Comment:
Looks like my naming conventions weren't as standard as I thought. 😄

Reply:
That amused me too =)